### PR TITLE
Spark 3.3: Support metadata column in the changelog table

### PR DESCRIPTION
**Pull request by flyrain**

### CODE REVIEW

**Comment by flyrain** [comment]
cc @aokolnychyi @RussellSpitzer @szehon-ho

File: spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestChangelogTable.java [hunk 1]
```diff
@@ -260,6 +260,14 @@ public void testManifestRewritesAreIgnored() {
         sql("SELECT id, _change_type FROM %s.changes ORDER BY id", tableName));
   }
 
+  @Test
+  public void testMetadataColumns() {
```
Comment (aokolnychyi) [inline spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestChangelogTable.java:264]: Can we add test for all metadata column types?

**Review by aokolnychyi — APPROVED** [review]
The implementation looks good to me but we need tests for each metadata column type.

File: spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestChangelogTable.java (hunk 1 above)
Comment (flyrain) [reply spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestChangelogTable.java:264]: Added in the new commit.

**Comment by flyrain** [comment]
Thanks for the review @aokolnychyi !

**Pull request merged**